### PR TITLE
ENS documentation

### PR DESCRIPTION
**Pull request by nivida**

### CODE REVIEW

File: docs/web3-eth-ens.rst [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+    > 0xfB6916095ca1df60bB79Ce92cE3Ea74c37c5d359
+
+------------------------------------------------------------------------------
+
+setAddress
+=====================
+
+.. code-block:: javascript
+
+    web3.eth.ens.setAddress(name, address, from);
```
Comment (frozeman) [inline docs/web3-eth-ens.rst]: the last parameter should be `option`

File: docs/web3-eth-ens.rst [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+    web3.eth.ens.setAddress(
+        'ethereum.eth',
+        '0xfB6916095ca1df60bB79Ce92cE3Ea74c37c5d359',
+        {
+            from: '0x9CC9a2c777605Af16872E0997b3Aeb91d96D5D8c'
+        }
+    ).then(function (result) {
+             console.log(result.events);
+    });
+    > AddrChanged(...)
```
Comment (frozeman) [inline docs/web3-eth-ens.rst:143]: How can one listen to the AddrChanged event?
Is there a `web3.ens.on('AddrChanged', listener)`?

File: docs/web3-eth-ens.rst [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+    }
+
+------------------------------------------------------------------------------
+
+setPubkey
+=====================
+
+.. code-block:: javascript
+
+    web3.eth.ens.setPubkey(name, x, y, from);
```
Comment (frozeman) [inline docs/web3-eth-ens.rst]: same here

File: docs/web3-eth-ens.rst [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+    > "0x0000000000000000000000000000000000000000000000000000000000000000"
+
+------------------------------------------------------------------------------
+
+setContent
+=====================
+
+.. code-block:: javascript
+
+    web3.eth.ens.setContent(name, hash, from);
```
Comment (frozeman) [inline docs/web3-eth-ens.rst]: same.

File: docs/web3-eth-ens.rst [hunk 5]
```diff
@@ -0,0 +1,431 @@
...
+    > 'QmXpSwxdmgWaYrgMUzuDWCnjsZo5RxphE3oW7VhTMSCoKK'
+
+------------------------------------------------------------------------------
+
+setMultihash
+=====================
+
+.. code-block:: javascript
+
+    web3.eth.ens.setMultihash(name, hash, from);
```
Comment (frozeman) [inline docs/web3-eth-ens.rst]: same.

File: docs/web3-eth-ens.rst (hunk 2 above)
Comment (nivida) [reply docs/web3-eth-ens.rst:143]: This is possible with: 
```

web3.eth.ens.registry.resolver(name).then(function(resolver){
    resolver.events.AddrChanged(options, callback);
});

```

Comment (frozeman) [reply docs/web3-eth-ens.rst:143]: Maybe that should be documented somewhere? maybe in the examples section

**Review by frozeman — APPROVED** [review]

**Pull request merged**